### PR TITLE
Fix #1911

### LINKED ISSUE #1911: No upload at charging if battery is on 100%
Hi! :)

I set 2 folders to auto upload when phone is charging. Now i recognised, if the phone is plugged in to the charger and the battery is below 100% it is working fine. But if the battery is at 100% (and the phone still on the charger) it is not working and the app/the upload is waiting for charging.

Could it be possible, that the app does not recognise the charger if the phone battery is on 100%?

Best regards,
HANNES 

### Environment data
Android version: 7.0
Device model: Huawei Mate 10 lite
Nextcloud app version: 2.0.0
Nextcloud server version: 12.0.3

### PR DESCRIPTION
**Pull request by mario**
Signed-off-by: Mario Danic <mario@lovelyhq.com>

### CODE REVIEW

File: src/main/java/com/owncloud/android/files/services/FileUploader.java [hunk 1]
```diff
@@ -942,7 +943,7 @@ public void onTransferProgress(long progressRate, long totalTransferredSoFar,
                         cancel(mCurrentUpload.getAccount().name, mCurrentUpload.getFile().getRemotePath()
                                 , ResultCode.DELAYED_FOR_WIFI);
                     } else if (mCurrentUpload.getIsChargingRequired() &&
-                            !Device.getBatteryStatus(MainApp.getAppContext()).isCharging()) {
+                            (!Device.getBatteryStatus(MainApp.getAppContext()).isCharging())) {
```
Comment (tobiasKaminsky) [inline src/main/java/com/owncloud/android/files/services/FileUploader.java]: Those parenthesis are not needed, or?

**Comment by tobiasKaminsky** [comment]
👍 
One minor question.

File: src/main/java/com/owncloud/android/files/services/FileUploader.java (hunk 1 above)
Comment (mario) [reply src/main/java/com/owncloud/android/files/services/FileUploader.java]: True.

**Comment by mario** [comment]
Merging with one approval.

**Pull request merged**